### PR TITLE
Improved TOA Compatibility

### PR DESCRIPTION
**Pull request by jojojo8359**
-Changed all TOA API links
-Removed station data on its own, uses station data retrieved from match data

### CODE REVIEW

File: simpleTest.py [hunk 1]
```diff
@@ -470,26 +494,26 @@ def __calculatePowerScore(self,teams, matches):
 # Main function ... reads the command line parms and starts up the UI if things look OK
 def main():
 
-    externalScoring = ExternalScoring("http://moftcscores.net/relic-recovery/event/q4","","")
+    externalScoring = ExternalScoring("https://theorangealliance.org/events/1819-MA-MAQ7", "https://theorangealliance.org/events/1819-MA-MAQ7", "ef4028f2eb076157f355763e7c6020859d923f03191731037f810dd0a3ba13aa")
```
Comment (kenpugsley) [inline simpleTest.py]: Probably not a good idea to put an api key here.  In fact ... perhaps we should remove this simpleTest file completely.  Not sure it really provides much value.

Comment (penrods) [reply simpleTest.py]: One potential solution that we use is saving tokens in environment varaibles.  If you setup things like Jenkins for continuous integration testing it is easy to set env variables without sharing all your secrets.  Also easy to setup on your local machine.

I've started the practice of creating a dev_setup.sh script that walks new devs through the process of setting things up.  E.g. checking for needed packages, env variables, etc and prompting them to pick an option or enter anything that might be needed.  Probably overkill for this for now, but a though for the future.

Comment (jojojo8359) [reply simpleTest.py]: I also agree that simpleTest could be removed. Environment variables sounds like a good option for storing those api keys. Like @penrods mentioned, a setup script sounds like a good solution for this, as well as checking that all the packages are installed, etc. Should be pretty easy to set up.

One other thing that I noticed while looking at the algorithm in this program is that it does not quite resemble the algorithm used in the [spreadsheet](https://docs.google.com/spreadsheets/d/1kYJcDS74AON5-Yz_hCjDnpyOBOUCW-75iaUy7dmo8Ik/edit?usp=sharing). Would it make sense to switch the program over to calculate Powerscores using the spreadsheet algorithm, or should the program version of the algorithm be left alone?

Comment (jojojo8359) [reply simpleTest.py]: @kenpugsley

**Pull request merged**